mxcubeweb/core/components/user/usermanager.py

```diff
@@ -202,7 +194,10 @@ def signout(self):
         self.app.server.user_datastore.deactivate_user(user)
         flask_security.logout_user()
 
-        self.emit_observers_changed()
+        self.app.server.emit(
+            "userChanged", room=user.socketio_session_id, namespace="/hwr"
```

Comment:
Send `userChanged` only to user who as been signed out.

Reply:
This was the culprit. Notifying the user who just signed out doesn't make sense, since the user is no longer listening for socket messages. Also, the UI already refetches the `login` state, obviously.

Reply:
Ok